features/FEATURE_LWIP/lwip-interface/lwip/src/netif/ppp/lwip_utils.c

```diff
@@ -605,7 +605,11 @@ void ppp_print_string(const u_char *p, int len, void (*printer) (void *, const c
  * ppp_logit - does the hard work for fatal et al.
  */
 static void ppp_logit(int level, const char *fmt, va_list args) {
+#ifdef PPP_LOGIT_BUFSIZE
```

Comment:
We made PPP_LOGIT_BUFSIZE configurable through lwipopts.h
It saves 512 bytes of RAM. Please note that ublox C027 have 32K RAM available.